### ISSUE TITLE
Altitudehold should be considered an autonomous mode for low throttle purposes

### ISSUE THREAD

**Issue by mlyle** [TauLabs/TauLabs#1912]

**Comment by kubark42** [comment]
> Therefore very long descents / landing sequences could do bad things.

Yes they do. And it leads to really startled looks on an intern's face followed by a "What did you do???". Ask me how I know. ;)
